reference-transaction-format.html

```diff
@@ -1028,6 +1028,7 @@ <h2 id="escrowfinish">EscrowFinish</h2>
 <li>If the corresponding <a href="#escrowcreate">EscrowCreate transaction</a> specified a <code>FinishAfter</code> time that is after the close time of the most recently-closed ledger, the EscrowFinish transaction fails.</li>
 <li>If the corresponding <a href="#escrowcreate">EscrowCreate transaction</a> specified a <code>CancelAfter</code> time that is before the close time of the most recently-closed ledger, the EscrowFinish transaction fails.</li>
 </ul>
+<p class="devportal-callout note"><strong>Note:</strong> The <a href="concept-transaction-cost.html">transaction cost</a> to submit an EscrowFinish transaction is higher if it contains a fulfillment. Instead of the usual base cost of 10 <a href="reference-rippled.html#specifying-currency-amounts">drops of XRP</a>, an EscrowFinish transaction costs 42 drops plus another 1 drop for every 16 bytes in size of the preimage. (For example, a fulfillment with a 32-byte preimage would cost 44 drops of XRP.)</p>
```

Comment:
For reference, this is just generated from the same text as the other comment. (The HTML files are all sourced from the markdown, so no need to comment on the HTML versions.)